swift/Arrow/Sources/Arrow/ArrowReader.swift

```diff
@@ -216,7 +216,67 @@ public class ArrowReader { // swiftlint:disable:this type_body_length
         return .success(RecordBatch(arrowSchema, columns: columns))
     }
 
-    public func fromStream( // swiftlint:disable:this function_body_length
+    public func fromMemoryStream( // swiftlint:disable:this function_body_length
```

Comment:
Could you add a comment that explains the difference between `fromMemoryStream` and `fromFileStream`?